File: js/0771-jewels-and-stones.js
/**
 * @param {string} jewels
 * @param {string} stones
 * @return {number}
 */
var numJewelsInStones = function (jewels, stones) {
  const jSet = new Set(jewels);
  return stones.split("").reduce((acc, curr) => acc + jSet.has(curr), 0);
};

module.exports = { numJewelsInStones };

File: go/solutions/0771-jewels-and-stones.go
package solutions

func NumJewelsInStones(jewels string, stones string) int {
	set := [256]int{}
	for i := range jewels {
		set[jewels[i]]++
	}

	ans := 0
	for i := range stones {
		if set[stones[i]] != 0 {
			ans++
		}
	}

	return ans
}
